bin/update_haproxy

```diff
@@ -4,8 +4,7 @@ x=0
 
 while [ $x -le 1 ]
 do
-    etcdctl --endpoint ${ETCD_PROTOCOL}://${ETCD_CLUSTER}:${ETCD_PORT} mkdir "/${HAPROXY_KEY}/${CLUSTER}/services"
-    etcdctl --endpoint ${ETCD_PROTOCOL}://${ETCD_CLUSTER}:${ETCD_PORT} set "/${HAPROXY_KEY}/${CLUSTER}/services/app/upstreams/${HNAME}" "${DOCKER_IP}" -ttl 10
+    etcdctl --endpoint ${ETCD_PROTOCOL}://${ETCD_CLUSTER}:${ETCD_PORT} set "/${HAPROXY_KEY}/${CLUSTER}/services/app/upstreams/${HNAME}" "${DOCKER_IP}" -ttl 10 >/dev/null
```

Comment:
this just displays the value that we place inside the key. At a 1 second interval so it makes it very hard to trace any other logs.

I agree that hiding the errors on this is not ideal. I need to find a sensible way to show errors but not the default activity